File: spec/code_samples/Node_request/list@check_email@{email}/get.js
var request = require("request");

var options = { method: 'GET',
  url: 'http://example.com/api/v1/list/check_email/%7Bemail%7D',
  qs: { username: 'SOME_STRING_VALUE' },
  headers:
   { authorization: 'Bearer REPLACE_BEARER_TOKEN',
     'content-type': 'application/json' } };

request(options, function (error, response, body) {
  if (error) throw new Error(error);

  console.log(body);
});

File: spec/code_samples/C_libcurl/lists@cleaned/get.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "GET");
curl_easy_setopt(hnd, CURLOPT_URL, "http://example.com/api/v1/lists/cleaned?username=SOME_STRING_VALUE");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "authorization: Bearer REPLACE_BEARER_TOKEN");
headers = curl_slist_append(headers, "content-type: application/json");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

CURLcode ret = curl_easy_perform(hnd);